hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java

```diff
@@ -18,6 +18,8 @@
 
 package org.apache.hadoop.fs.s3a;
 
+import io.netty.util.internal.StringUtil;
```

Comment:
-1 to any dependency on netty in existing classes

Reply:
ack